packages/cli/lib/lib/webpack/create-load-manifest.js

```diff
@@ -34,7 +41,10 @@ module.exports = assets => {
     obj = Object.assign({}, defaults);
     obj[filename] = { type:'script', weight:0.9 };
     if (css) obj[css] = { type:'style', weight:0.9 };
-    path = filename.replace(/route-/, '/').replace(/\.chunk(\.\w+)?\.js$/, '').replace(/\/home/, '/');
+    path = filename.replace(/route-/, '/')
+      .replace(/\.chunk(\.\w+)?\.js$/, '')
```

Comment:
```
.replace(/\.chunk(\.\w+)?(\.esm)?\.js$/, '')
```

Reply:
this one isn't working hence keeping as is

Reply:
nevermind... fixed it